testing/run-scripts/run_cloud.sh

```diff
@@ -175,14 +178,15 @@ echo "CLOUD_INSTALL_STATUS_INSTALLING_SSH"
 echo "CLOUD_INSTALL_PROGRESS 60"
 if ! docker ps -a | grep uproxy-sshd >/dev/null; then
   if ! docker images | grep uproxy/sshd >/dev/null; then
-    TMP_DIR=/tmp/uproxy-sshd
-    rm -fR $TMP_DIR
-    cp -R ${BASH_SOURCE%/*}/../../sshd/ $TMP_DIR
-
+    TMP_DIR=`mktemp -d`
     echo -n "$BANNER" > $TMP_DIR/banner
     echo -n "$PUBLIC_IP" > $TMP_DIR/hostname
 
-    docker build -t uproxy/sshd $TMP_DIR
+    docker run --restart=always --name uproxy-sshd $SSHD_IMAGE
```

Comment:
I don't think you want two `docker run` statements - there's another at line 195.